packages/contracts/contracts/core/plugin/AragonPlugin.sol

```diff
@@ -0,0 +1,30 @@
+pragma solidity 0.8.10;
+
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
```

Comment:
Shouldn't this be `import "@openzeppelin/contracts/proxy/utils/Initializable.sol";` instead?